autoload/neomake/makers/ft/typescript.vim

```diff
@@ -10,6 +10,7 @@ function! neomake#makers#ft#typescript#tsc() abort
         \ 'args': ['--project', neomake#utils#FindGlobFile('tsconfig.json'), '--noEmit', '--watch', 'false'],
         \ 'append_file': 0,
         \ 'errorformat':
+            \ '%E%f(%l\,%c): error %m,' .
             \ '%E%f %#(%l\,%c): error %m,' .
```

Comment:
The new errorformat is the same as the previous one below it basically.
` %#` translates to "at least one space".
Why is the new one needed?